ophyd/pseudopos.py

```diff
@@ -227,22 +246,34 @@ def connected(self):
         return all(mtr.connected for mtr in self._real)
 
     def stop(self):
-        for pos in self._real:
-            pos.stop()
-
-        Positioner.stop(self)
+        try:
+            del self._move_queue[:]
+            for pos in self._real:
+                pos.stop()
```

Comment:
picky, try needs to be inside or will only make it as far as the first failing positioner